drulenium_visual_testing/README.md

```diff
@@ -0,0 +1,47 @@
+# Visual Regression Testing via Drulenium #
+
+This example will show you how you can automatically run visual tests on your pantheon hosted environments using Drulenium suite of modules & libraries. 
+
+This will allow you to do a comparative visual diff between the test environment and the development environemnt(including Multidev environments) everytime you push code to the development environment.
+
+As a note, Tests run on https://travis-ci.org/ and is free to use for unlimited number of pages but the test results are public. For more information read http://drulenium.org/how-to-automate-testing-in-pantheon-hosted-website
+
+## Instructions ##
+
+Setting up this example is easy:
+
+1. Add the example post_drulenium_github.php script to the 'private/scripts/' directory of your code repository.
+2. Add the secret github account parameters into a file called secrets.json and store it in the private files directory of Test & Live environments and Clone files from the test environment into other dev environments.
+
+  ```json
+    {
+      "github_username": "Drulenium",
+      "github_repository": "pantheon-travis",
+      "github_accesstoken": "ddedc2530c38734ea00651b80d41de5f1d2a64df",
```

Comment:
@ataylorme It's a test account and I am good with giving access to write into it for users who want to try this before setting up their own github account, travis account, Imgur.com account and configuring them accordingly.

This is the shared github account method of testing as described at http://drulenium.org/how-to-automate-testing-in-pantheon-hosted-website
![image](https://cloud.githubusercontent.com/assets/545961/14711069/f0f152ea-07a5-11e6-81c4-532234c831f5.png)